entrypoint.sh

```diff
@@ -115,20 +115,27 @@ main() {
   aws --version
 
   # Iterate over $INPUT_SOURCE multiline string and run aws s3 $COMMAND
-  while IFS= read -r source; do
+while IFS= read -r source; do
     if [ -n "$source" ]; then
-        if [ "$COMMAND" == "cp" ] || [ "$COMMAND" == "mv" ] || [ "$COMMAND" == "sync" ]
-        then
-            echo "aws s3 $COMMAND \"$source\" $INPUT_DESTINATION $INPUT_FLAGS"
-            aws s3 "$COMMAND" "$source" "$INPUT_DESTINATION" "$INPUT_FLAGS"
+        if [ "$COMMAND" == "cp" ] || [ "$COMMAND" == "mv" ] || [ "$COMMAND" == "sync" ]; then
```

Comment:
Just a suggestion - let's [replace spaces with underscores.](https://stackoverflow.com/a/19661428) Should eliminate the majority of cases.
```suggestion
        source="${source// /_}"
        if [ "$COMMAND" == "cp" ] || [ "$COMMAND" == "mv" ] || [ "$COMMAND" == "sync" ]; then
```

Reply:
@timofei-kormachev-pipedrive I believe this should be done in the test framework level when files are generated - if the file names stay with spaces and we convert the spaces in source to underscores, the source path will not be recognized & artifacts won't be uploaded. I agree that the output json can be more consistent but it needs to be investigated where the change should be done